pylab/website/templates/website/project_details.html

```diff
@@ -14,9 +14,20 @@
   {% endif %}
 </div>
 
-<h1>{{ project.title }}</h1>
+<h1>{{ project.title }}
+  <small><span class="label label-default">{{ project.project_status }}</span></small>
```

Comment:
Do not use status value here, use status label: `project.get_project_status_display`.

See: https://docs.djangoproject.com/en/1.8/ref/models/instances/#django.db.models.Model.get_FOO_display